pkgs/applications/misc/huggle/01-extensions.patch

```diff
@@ -0,0 +1,112 @@
+index f459515..b97ada6 100644
+--- a/src/extensions/extension-scoring/CMakeLists.txt
++++ b/src/extensions/extension-scoring/CMakeLists.txt
+@@ -76,5 +76,5 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_scoring DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_scoring DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_scoring DESTINATION @out@/extensions)
+ endif()
+Submodule src/extensions/extension-splitter-helper contains modified content
+diff --git a/src/extensions/extension-splitter-helper/CMakeLists.txt b/src/extensions/extension-splitter-helper/CMakeLists.txt
+index d56f8a5..46ede3e 100644
+--- a/src/extensions/extension-splitter-helper/CMakeLists.txt
++++ b/src/extensions/extension-splitter-helper/CMakeLists.txt
+@@ -67,5 +67,5 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_sh DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_sh DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_sh DESTINATION @out@/extensions)
+ endif()
+Submodule src/extensions/extension-thanks contains modified content
+diff --git a/src/extensions/extension-thanks/CMakeLists.txt b/src/extensions/extension-thanks/CMakeLists.txt
+index f4bfa1b..eea28eb 100644
+--- a/src/extensions/extension-thanks/CMakeLists.txt
++++ b/src/extensions/extension-thanks/CMakeLists.txt
+@@ -67,6 +67,6 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_thanks DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_thanks DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_thanks DESTINATION @out@/extensions)
+ endif()
+ 
+Submodule src/extensions/mass-delivery contains modified content
+diff --git a/src/extensions/mass-delivery/CMakeLists.txt b/src/extensions/mass-delivery/CMakeLists.txt
+index 29c52fa..189b06d 100644
+--- a/src/extensions/mass-delivery/CMakeLists.txt
++++ b/src/extensions/mass-delivery/CMakeLists.txt
+@@ -75,5 +75,5 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_md DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_md DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_md DESTINATION @out@/extensions)
+ endif()
+diff --git a/src/huggle_core/definitions_prod.hpp b/src/huggle_core/definitions_prod.hpp
+index 1fc5b1fb..17881ade 100644
+--- a/src/huggle_core/definitions_prod.hpp
++++ b/src/huggle_core/definitions_prod.hpp
+@@ -216,7 +216,7 @@ namespace std { typedef decltype(nullptr) nullptr_t; }
+     #define HUGGLE_GLOBAL_EXTENSION_PATH            QCoreApplication::applicationDirPath() + "/extensions"
+ #elif defined __linux__
+     #define HUGGLE_UPDATER_PLATFORM_TYPE            "linux"
+-    #define HUGGLE_GLOBAL_EXTENSION_PATH            "/usr/local/share/huggle/extensions"
++    #define HUGGLE_GLOBAL_EXTENSION_PATH            "@out@/extensions"
```

Comment:
Is it a thing that you can install more extensions afterwards? This would break with this

Reply:
People could install more extensions, but 1. this does not have many extensions and most of them come bundled with this, 2. there is a per-user directory for extensions already, and 3. I don't see a specific need for installing extensions system-wide when per-user works and is configurable.